pkg/services/auth/auth_token.go

```diff
@@ -214,7 +221,14 @@ func (s *UserAuthTokenService) TryRotateToken(ctx context.Context, token *models
 
 	s.log.Debug("token needs rotation", "tokenId", model.Id, "authTokenSeen", model.AuthTokenSeen, "rotatedAt", rotatedAt)
 
-	clientIP = util.ParseIPAddress(clientIP)
+	clientIP, err := util.ParseIPAddress(clientIP)
+	if err != nil {
+		s.log.Debug("Failed to parse client IP address", "clientIp", clientIP, "err", err)
+		clientIP = ""
```

Comment:
Assuming that we want to be tolerant here and just go with an empty client IP address.

Reply:
Same comment as earlier

Reply:
Thanks, fixed.